src/core/worker.js

```diff
@@ -106,6 +106,22 @@ var WorkerMessageHandler = PDFJS.WorkerMessageHandler = {
 
           var length = fullRequestXhr.getResponseHeader('Content-Length');
           length = parseInt(length, 10);
+//#if !(FIREFOX || MOZCENTRAL)
```

Comment:
Wouldn't it make more sense to use `(GENERIC || CHROME)` instead? That'd be more explicit than listing everything that should not be included.